lib/agent.js

```diff
@@ -815,7 +815,9 @@ Agent.prototype._handlePresenceData = function(presence) {
   var agent = this;
   backend.trigger(backend.MIDDLEWARE_ACTIONS.sendPresence, this, context, function(error) {
     if (error) {
-      return agent.send({a: 'p', ch: presence.ch, id: presence.id, error: getReplyErrorObject(error)});
+      if (backend.sendPresenceMiddlewareServerErrors) backend.emit('error', error);
```

Comment:
From verbal discussion today:
1) Rename flag to something like `dontForwardSendPresenceErrorToClient`
2) New non-fatal error handling mechanism - Pass in an error handler to Backend. If it's not passed in, we have a default that just logs to console.
3) Use that error handling for this case, later we can convert more of the console logs in the backend code to use the new error handler.